File: Bookworm/app/src/main/java/com/briatka/pavol/bookworm/customobjects/Book.java
package com.briatka.pavol.bookworm.customobjects;

import org.simpleframework.xml.Element;
import org.simpleframework.xml.Root;

@Root(name = "book", strict = false)
public class Book {

    @Element(name = "title")
    private String title;

    @Element(name = "average_rating")
    private String rating;

    @Element(name = "reviews_widget", data = true)
    private String reviews;

    public String getTitle() {
        return title;
    }

    public String getRating() {
        return rating;
    }

    public String getReviewsWidget() {
        return reviews;
    }
}

File: Bookworm/app/src/main/java/com/briatka/pavol/bookworm/customobjects/Book.kt
package com.briatka.pavol.bookworm.customobjects

import org.simpleframework.xml.Element
import org.simpleframework.xml.Root

@Root(name = "book", strict = false)
class Book @JvmOverloads constructor(
        @field:Element(name = "title")
        @param:Element(name = "title")
        val title: String? = null,
        @field:Element(name = "average_rating")
        @param:Element(name = "average_rating")
        val rating: String? = null,
        @field:Element(name = "reviews_widget", data = true)
        @param:Element(name = "reviews_widget", data = true)
        val reviewsWidget: String? = null

) {}
